shared/oval/sysctl_kernel_exec_shield.xml

```diff
@@ -10,13 +10,12 @@
       <description>The kernel runtime parameter "kernel.exec-shield" should not be disabled and set to 1 on 32-bit systems.</description>
     </metadata>
     <criteria operator="OR">
-      <criteria operator="AND" comment="system is RHEL6">
-        <extend_definition comment="RHEL6 installed" definition_ref="installed_OS_is_rhel6" />
-        <criterion comment="kernel runtime parameter kernel.exec-shield set to 1" test_ref="test_runtime_sysctl_kernel_exec_shield" />
-        <criterion comment="kernel /etc/sysctl.conf parameter kernel.exec-shield set to 1" test_ref="test_static_sysctl_kernel_exec_shield" />
-      </criteria>
-      <criteria operator="AND">
-        <extend_definition comment="32-bit system" definition_ref="system_info_architecture_x86" />
+      <criteria operator="AND" comment="system is RHEL6-based, is x86-32, then eval kernel.exec-shield">
+        <criteria operator="OR" comment"system is RHEL6 or CentOS6">
```

Comment:
Platform should be changed to `multi_platform_rhel`.

And this same file claims to be gererated but I could not find its template, so I think we can remove that claim.